python/monarch/gradient/_gradient_generator.cpp

```diff
@@ -420,12 +426,20 @@ struct GradientGenerator {
     DEBUG_PRINT(
         "// add: " << node->node->name()
                    << ", input_nr=" << static_cast<int>(input_nr) << "\n");
+#if TORCH_VERSION_NEWER_THAN(2, 8, 0)
```

Comment:
@pzhan9 Shouldn't this condition have been for 2.9.1? Even the release/2.9 branch has a 4-argument version: https://github.com/pytorch/pytorch/blob/d38164a545b4a4e4e0cf73ce67173f70574890b6/torch/csrc/autograd/input_buffer.h#L31